ocs_ci/utility/environment_check.py

```diff
@@ -44,7 +44,11 @@ def compare_dicts(before, after):
 
     Returns:
         list: List of 2 lists - ('added' and 'removed' are lists)
+        None: If either parameter is None
     """
+    if not before or not after:
+        return None
```

Comment:
We should probably log when this is the case

Reply:
Coady, thanks for the review. I added a log statement.